domains/nidun.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "new update to request new cname",
+    "repo": "https://github.com/nldun/ininidun.github.io",
+    "owner": {
+        "username": "nldun",
+        "email": "rezkyto@gmail.com",
+    },
+    "record": {
+        "CNAME": "ininidun.github.io"
```

Comment:
website doesnt exist (404 page)

Reply:
tho if you do have website hosted post the link here and if its good ill merge it